src/docfx/build/toc/ServicePageGenerator.cs

```diff
@@ -85,7 +85,7 @@ internal class ServicePageGenerator
                     }
                     else
```

Comment:
This else could be non-empty `childHref` and link type is not `RelativePath`.
For this case, shall we set `childHref` as `null` as well?

Reply:
No. It should be correct now since the last PR is merged.............🤦‍😔